src/App.js

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect, useState } from 'react';
+import InputRange from 'react-input-range';
+import 'react-input-range/lib/css/index.css'
+import logo from './jurgisLogo.png';
+import bgimage from './dp.jpg'
+import MovieCard from './components/MovieCard.js';
+import Spinner from 'react-bootstrap/Spinner'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+import { Jumbotron, Button, Navbar, Nav, NavDropdown, FormControl, } from "react-bootstrap";
+import ReactModal from 'react-modal';
+import YouTube from '@u-wave/react-youtube';
+
+let apikey = process.env.REACT_APP_JURGIS;
+let keyword = '';
+let page = 1;
+let movieList = [];
+
+function App() {
```

Comment:
Your overall app structure could use a little bit more refinement here. You've jammed everything in one component, which makes your app harder to maintain in the long run.

Perhaps a better structure would've been:
```
<App>
   <Filters/>
   <MovieList>
        ... <Movie> repeated
   </MovieList/>
</App>
```
Something like that.